config.yml

```diff
@@ -0,0 +1,67 @@
+# General
+serverName: UUID
+colors: true
+showLogo: true
+logLevel: INFO
+
+# Connectivity
+webServerEnabled: true
+tcpServerEnabled: true
+port: 6020
+host: 0.0.0.0
+tcpPort: 6021
+tcpHost: 0.0.0.0
+httpServer: null
+urlPath: /engine.io
+
+# SSL Configuration
+sslKey: null
+sslCert: null
+sslCa: null
+
+# Data Manipulation
+dataTransforms: null
```

Comment:
We might also not want to expose this via the yaml file since it expects JS functions and array structure, meaning whoever uses it will probably be doing it via code